src/main/webapp/creatives/constants.js

```diff
@@ -100,3 +100,14 @@ export const CreativeErrorCode = {
   MESSAGES_NOT_FOLLOWING_SPEC: 1110,
   PLAYER_RESPONSE_TIMEOUT: 1111,
 };
+
+/** A list of server routes that keep track of each interaction when fetched. */
+export const Urls = {
```

Comment:
Make sure to also remove this from utils.js

Reply:
Yeah that should be in my other PR